cmd/integrationArtifactDeploy.go

```diff
@@ -92,14 +94,135 @@ func runIntegrationArtifactDeploy(config *integrationArtifactDeployOptions, tele
 		log.Entry().
 			WithField("IntegrationFlowID", config.IntegrationFlowID).
 			Info("successfully deployed into CPI runtime")
-		return nil
+		error := PollIFlowDeploymentStatus(retryCount, config, httpClient)
```

Comment:
error is the name of the error interface. Better choose another name.

Reply:
fixed